tensorflow/python/ops/math_ops.py

```diff
@@ -3657,6 +3685,15 @@ def unsorted_segment_mean(data, segment_ids, num_segments, name=None):
 
   If the given segment ID `i` is negative, the value is dropped and will not
   be added to the sum of the segment.
+  
+  For example:
+  >>> x = tf.constant([1, 2, 3, 4])
```

Comment:
Add empty line above.

Reply:
Empty line added where it shouldn't have. It should be between `For example` and the doctest